sunpy/map/mapbase.py

```diff
@@ -1598,24 +1598,22 @@ def save(self, filepath, filetype='auto', **kwargs):
         Parameters
         ----------
         filepath : `str`
-            Location to save file to.
+            Location to save the file to. The file extension determines the format.
         filetype : `str`, optional
-            Any supported file extension, defaults to ``"auto"``.
+            The file format to save the map in. Defaults to ``"auto"`` which infers
+            the format from the file extension. Supported formats include FITS, JP2, and ASDF.
```

Comment:
We don't support `save("wibble", filetype="asdf")` which this docstring implies we would, so I think we should add that (see comment below).